tools/perf/configs/dashboard/smoke-brave.json5

```diff
@@ -35,6 +35,13 @@
         "load:media:youtube:2018",
       ]
     },
+    {
+      "name": "system_health.memory_desktop",
+      "pageset-repeat": 10,
```

Comment:
```suggestion
      "pageset-repeat": 1,
```

Reply:
It's just a test file, pageset-repeat should be 1 everywhere.

Reply:
Okay, Got it!